doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -373,42 +382,39 @@ For the purpose of this guide we will use a fairly abstract server-side profile
 }
 ```
 
-### Setting server-side endpoint profile schema from Admin UI
+### Setting server-side EP profile schema from Administration UI
 
-The list of server-side endpoint profile schemas created by a tenant developer for the application is shown in the Server-side EP profile schemas window, which can be 
-opened from the application menu on the navigation panel as illustrated by the following screenshot.
+To view the list of server-side endpoint profile schemas created by a [tenant developer]({{root_url}}Glossary/#tenant-developer), open the the **Server-side EP profile** page under the **Schemas** section of the application.
 
 <img src="admin-ui/Server-side endpoint profile schema.png">
 
-To export the server-side EP profile schema, click Export in the last column of desired schema row and select export method from drop-down. See
-[CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support) for available export methods details.
-As a tenant developer, you can create new server-side EP schemas for the application as follows:
+To export a client-side EP profile schema, click the corresponding **Export** button and choose the export method from the drop-down list.
+See [CT schema export support]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library/#ct-schema-export-support) for available export methods details.
+
+As a tenant developer, you can create new server-side EP schemas for your application as follows:
+
+1. Under the **Schemas** section of the application, click **Server-side EP profile schemas**, then click the **Add schema** button.
+
+2. On the **Add profile schema** page, enter your schema name and description (optional).
 
-1. In the **Server-side EP profile schemas** window for the application, click **Add schema**.
-2. In the **Add profile schema** window enter the name of the schema.
 3. Then create a schema using one of the two options:
 
-    1. Using the existing CT by clicking **Select existing type** and selecting exiting CT version from FQN and version drop-downs.
+    * Use the existing [common type (CT)]({{root_url}}Glossary/#common-type-ct) by clicking **Select existing type**.
```

Comment:
This is a 'legal' repetition, because here we're describing the server-side instruction that in some parts is similar to the client-side instructions.